holidays/countries/turks_and_caicos_islands.py

```
@@ -0,0 +1,81 @@
+        self._add_christmas_day(tr("Christmas Day"))
+
+        # Boxing Day
+        self._add_christmas_day_two(tr("Boxing Day"))
+
+
+class TC(TurksAndCaicosIslands):
+    pass
+
+
+class TCA(TurksAndCaicosIslands):
+    pass
```

Ruff (0.8.2)

No newline at end of file

Add trailing newline

(W292)